pkg/klusterlet/clusterinfo/clusterinfo_controller.go

```diff
@@ -121,6 +115,7 @@ func (r *ClusterInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			newStatus.ClusterID = value
 		case clusterclaim.ClaimOCMProduct:
 			newStatus.KubeVendor = r.getKubeVendor(value)
+			product = value
```

Comment:
I am not quite sure using this claim is the best approach, I mean if there is a new supported product(ex: microshift maybe), we should add the new product type in https://github.com/stolostron/multicloud-operators-foundation/pull/526/files#diff-ac6674a4f427b422665e2fbea90ad4e776f2946ae22cc9974e4eb9045d5a4312R452 right? and I think it's a little bit error-prone(easy to forget to change).

Is there always a claim to indicate it is OCP no matter what the OCP product is? `id.openshift.io` or `version.openshift.io`?